Dockerfile

```diff
@@ -21,12 +21,15 @@ RUN pecl install redis && \
 RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin \
     --filename=composer && hash -r
 
+# User and Group
+RUN groupadd -r -g 200 seat && useradd --no-log-init -r -g seat -u 200 seat
+
 # Install SeAT
 RUN cd /var/www && \
-    composer create-project eveseat/seat:5.0.x-dev --stability dev --no-scripts --no-dev --no-ansi --no-progress --prefer-source && \
+    composer create-project eveseat/seat:5.0.x-dev --stability dev --no-scripts --no-dev --no-ansi --no-progress && \
     composer clear-cache --no-ansi && \
     # Fix up the source permissions to be owned by www-data
-    chown -R www-data:www-data /var/www/seat && \
+    chown -R seat:seat /var/www/seat && \
     cd /var/www/seat && \
     # Setup the default configuration file
     php -r "file_exists('.env') || copy('.env.example', '.env');"
```

Comment:
What is this for? I

Reply:
Wait, it selected the wrong line. Apparently I can only comment on changes.

What I wanted to ask is why we have the .env setup. Isn't that provided from the outside?

Reply:
the `.env` inside `seat-docker` will be used as initial container seeding for environment variables and `docker-compose` files
this is the `.env` from the core - which might invole a little bit when the docker one will not - in such case, value are loaded this way :
1. config
2. environment variable
3. `.env` file

> lower is overriden by upper